ideascube/blog/tests/test_search.py

```diff
@@ -12,16 +12,19 @@ def test_nothing_is_indexed_without_any_fixture():
     assert Content.objects.count() == 0
 
 
+@pytest.mark.usefixtures('cleansearch')
```

Comment:
Maybe move that in a `pytest_setup` conftest.py hook not to repeat it again and again?
Not sure it's good idea because: it's less explicit, it will consume a bit more time (because will be run at every test).
But let's think about it.